docs/docs/config/behaviors.md

```diff
@@ -9,6 +9,12 @@ See [Configuration Overview](index.md) for instructions on how to change these s
 
 See the [zmk/app/dts/behaviors/](https://github.com/zmkfirmware/zmk/tree/main/app/dts/behaviors) folder for all default behaviors.
 
+## Kconfig
+
+| Config                             | Type | Description                                          | Default |
+| ---------------------------------- | ---- | ---------------------------------------------------- | ------- |
+| `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE`  | int  | The size of the behaviors queue                      | 64      |
```

Comment:
```suggestion
| `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE`  | int  | Maximum number of behaviors to allow queueing from a macro or other complex behavior | 64      |
```

The descriptions in these docs match the description in the Kconfig file. Note that you need to run `prettier` again to format the table properly after this change.

Reply:
done!